sqlmesh/dbt/builtin.py

```diff
@@ -249,6 +256,21 @@ def source(package: str, name: str) -> t.Optional[BaseRelation]:
     return source
 
 
+def generate_model(model: AttributeDict, raw_code: str) -> Model:
+    class Model:
+        def __init__(self, model: AttributeDict) -> None:
+            self._model = model
+            self._raw_code_key = "raw_code" if DBT_VERSION >= (1, 3, 0) else "raw_sql"  # type: ignore
+
+        def __getattr__(self, key: str) -> t.Any:
+            if key == self._raw_code_key:
```

Comment:
I don't think this is necessary if you just do:
```
self._model = AttributeDict({
    **model,
    raw_code_key: raw_code,
})
```
in __init__. Actually if you don't really need the `Model` class at all if you just construct this dict and return it in `generate_model`.

Reply:
Good feedback, thank you! Addressed here: https://github.com/TobikoData/sqlmesh/pull/5269/commits/f8bd60b58deb457f7cfe48f85881bb945c2c277f.